src/main/java/core/Main.java

```diff
@@ -0,0 +1,67 @@
+package core;
+
+import core.basesyntax.file.DataConverter;
+import core.basesyntax.file.DataConverterImpl;
+import core.basesyntax.file.ReadFromFile;
+import core.basesyntax.file.ReadFromFileImpl;
+import core.basesyntax.file.ReportGenerator;
+import core.basesyntax.file.ReportGeneratorImpl;
+import core.basesyntax.file.ReportWriter;
+import core.basesyntax.file.ReportWriterImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.operations.BalanceOperationHandler;
+import core.basesyntax.operations.OperationHandler;
+import core.basesyntax.operations.PurchaseOperationHandler;
+import core.basesyntax.operations.ReturnOperationHandler;
+import core.basesyntax.operations.SupplyOperationHandler;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    private static final String inputFilePath = "src\\main\\resources\\inputValue.csv";
+    private static final String outputFilePath = "finalReport.csv";
```

Comment:
```suggestion
    private static final String INPUT_FILE_PATH = "src\main\resources\inputValue.csv";
    private static final String OUTPUT_FILE_PATH = "finalReport.csv";
```
use the correct constant name style